schema/CompoundSchema.mcf

```diff
@@ -578,3 +578,150 @@ description: "A reliability score is manually set for all genes and indicates th
 descriptionUrl: "https://www.proteinatlas.org/about/assays+annotation#if_reliability_score"
 rangeIncludes: dcs:HumanProteinOccurrenceReliabilityEnum
 domainIncludes: dcs:HumanProteinOccurrence
+
+Node: dcid:Antibody
+name: "Antibody"
+typeOf: schema:Class
+subClassOf: dcs:Protein
+description: "An antibody is a kind of protective protein which is produced by the immune system when substances from the outer source invade the living organisms. The process of producing the antibodies is called immune response."
+
+Node: dcid:Antigen
+name: "Antigen"
+typeOf: schema:Class
+subClassOf: dcs:ChemicalCompound
+description: "The invading substances from the outer source which causes the immune response. It refers to a molecule or molecule structure which may interact with the immune cell antigen receptor or antibody. "
+descriptionUrl:"https://en.wikipedia.org/wiki/Antigen"
+
+Node: dcid:recognizesAntigen
+typeOf: schema:Property
+name: "recognizesAntigen"
+description: "The targeting antigen of the antibody."
+rangeIncludes: dcs:Protein
+domainIncludes:dcs:Antibody
+# rangeIncludes proteins with links to UniProt, chemicals with IDs to ChEBI and a few samples to other databases (InterPro, NCBI, Prosite, UniProt-Subcell).
+
+Node: dcid:antigenType
+typeOf: schema:Property
+name: "antigenType"
+description: "The type of the antigen for an antibody. There are types as protein, chemical and the others. "
+rangeIncludes: dcs:Protein
+domainIncludes:dcs:Antibody
+
+Node: dcid:antibodyType
+typeOf: schema:Property
+name: "antibodyType"
+description: "The type of antibody, including bispecific antibody, nanobody and DARPins. "
+rangeIncludes: dcs:AntibodyTypeEnum 
+domainIncludes:dcs:Antibody
+
+Node: dcid:antibodyApplication
+typeOf: schema:Property
+name: "antibodyApplication"
+description: "The applications of the antibody in the experiments, such as ELISA, Immunofluorescence, Immunoprecipitation, etc."
+rangeIncludes: schema:Text
+domainIncludes:dcs:Antibody
+
+Node: dcid:epitope
+typeOf: schema:Property
+name: "epitope"
+description: "Also known as antigenic determinant. It is the recognizing part of an antigen by the interacting immune cell antigen receptor or the antibody."
+rangeIncludes: schema:Text
+domainIncludes:dcs:Antibody
+descriptionUrl:"https://en.wikipedia.org/wiki/Epitope"
+
+Node: dcid:abcdID
+typeOf: schema:Property
+name: "abcdID"
+description: "The accession identifier of the antibody in the ABCD antibody database. "
+rangeIncludes: schema:Text
+domainIncludes:dcs:Antibody
+descriptionUrl:"https://web.expasy.org/abcd/"
+
+Node: dcid:patentID
+typeOf: schema:Property
+name: "patentID"
+description: "The identifier of patent in PATENTSCOPE. Using PATENTSCOPE you can search 90 million patent documents including 3.9 million published international patent applications (PCT)."
```

Comment:
This seems too specific a description for a general sounding property like "patentID".

Does it make sense to make the property more specific, like "patentScopeID"?